Explore page

frozenhelium opened the pull request.

frozenhelium committed:
Add Explore page

frozenhelium committed:
Add Orders bar and checkout popup

tnagorra commented on app/views/HomePage/index.tsx:
```diff
@@ -128,9 +128,8 @@ function HomePage() {
-                                to="#"
-                                // TODO: implement page
                                 // TODO: use SmartButtonLikeLink
+                                to="/explore/"
```
Let's use SmartButtonLikeLink

tnagorra committed:
Separate out OrdersBar and OrdersModal

- Add functionality to add/remove book from wishlist

frozenhelium committed:
Add book detail modal

frozenhelium committed:
Unify BookItem

frozenhelium committed:
Add search and sort in explore page

tnagorra committed:
Handle error and loading on BookModal

- Add add/remove from wishlist
- Refetch order stats after order
- Remove "page size" change
